stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/querychannels/QueryChannelsController.kt

```diff
@@ -67,10 +70,39 @@ internal class QueryChannelsController(
             when {
                 loading -> QueryChannelsController.ChannelsState.Loading
                 channels.isEmpty() -> QueryChannelsController.ChannelsState.OfflineNoResults
-                else -> QueryChannelsController.ChannelsState.Result(channels)
+                else -> {
+                    val channelMutes = domainImpl.currentUser.channelMutes
```

Comment:
You shouldn't do any changes here. `channelState` is based on the `_sortedChannels` which one is based on the `_channels` property. We should make changes only in value of this `_channels`